### PR TITLE
#1017 RtForkTest.java:56-58: Apply this rule to other

### PR DESCRIPTION
**Pull request by tmjee**
TS/PDD #1017 
- fix indention 
- added todo puzzle to redesign RtBlobsTest and remove checkstyle

### CODE REVIEW

**Comment by dmarkov** [comment]
@tmjee Let me find a reviewer for this pull request, thanks for submitting it

**Comment by dmarkov** [comment]
@darkled it's yours, please review

File: src/test/java/com/jcabi/github/RtBlobsTest.java [hunk 1]
```diff
@@ -50,10 +51,20 @@
  * @author Alexander Lukashevich (sanai56967@gmail.com)
  * @version $Id$
  * @checkstyle MultipleStringLiteralsCheck (100 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```
Comment (darkled) [inline src/test/java/com/jcabi/github/RtBlobsTest.java]: @tmjee let's add a `@todo` puzzle to re-design this class and remove switching off checkstyle here

File: src/test/java/com/jcabi/github/RtBlobsTest.java [hunk 2]
```diff
@@ -63,8 +74,9 @@ public void canCreateBlob() throws Exception {
         final String content = "Content of the blob";
         final String body = blob().toString();
         final MkContainer container = new MkGrizzlyContainer().next(
-            new MkAnswer.Simple(HttpURLConnection.HTTP_CREATED, body)
-        ).next(new MkAnswer.Simple(HttpURLConnection.HTTP_OK, body)).start();
+                new MkAnswer.Simple(HttpURLConnection.HTTP_CREATED, body)
```
Comment (darkled) [inline src/test/java/com/jcabi/github/RtBlobsTest.java]: @tmjee indentations seems to be not correct here

**Comment by darkled** [comment]
@tmjee please see minor comment above. Also please change the PR according to our [quality rules](http://www.teamed.io/qa.html):
>- Pull request title explains the problem it is fixing.
>- Pull request description explains the solution proposed and contains a link to the original ticket it is related to.

**Comment by tmjee** [comment]
TS/PDD #1017 RtForkTest.java:56-58: Apply this rule to other
- fix indention issue 
- add puzzle

**Comment by tmjee** [comment]
@darkled Changes made, details updated. Thx

**Comment by darkled** [comment]
@tmjee great, thanks

**Comment by darkled** [comment]
@rultor merge pls

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by darkled** [comment]
@yegor256 there is some accidental error, could you try to merge once more, please?

**Comment by yegor256** [comment]
@darkled every time you see such an error, it's a good practice to create a ticket, complaining about tests instability.

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by darkled** [comment]
@tmjee please take a look at 
```
Tests in error: RtIssueITCase.talksInGithubProject:107 » ClassCast javax.json.JsonValue$1 cann...
```
It seems something is broken.

**Comment by tmjee** [comment]
@darkled Can I get more details about the error? Eg. a copy of the logs / stack traces for the test in /home/r/repo/target/failsafe-reports directory. Thx

**Comment by darkled** [comment]
@tmjee you can find all the information in the rultor's merge log, link is mentioned above.

**Comment by tmjee** [comment]
@darkled the log mentioned 

```
There are test failures. 
Please refer to /home/r/repo/target/failsafe-reports for the individual test results. 
```
I'd like to get that test `RtIssueITCase.talksInGithubProject` test results. It is not in rultor's merge log because that single CCE doesn't makes much sense looking at the code at that line.

**Comment by darkled** [comment]
@tmjee please see http://www.rultor.com/t/5241-143366877, I think it should be enough. Also I am not sure if we have access to `/home/r/repo/target/failsafe-reports`, because `rultor` uses docker container for every build.

**Comment by tmjee** [comment]
@darkled see #1168 for more info. This is another separate issue. Please try run merge again and it should work.

**Comment by darkled** [comment]
@tmjee ok, then please ignore this test and let's try to merge once more

**Comment by darkled** [comment]
@tmjee is there any news?

**Comment by tmjee** [comment]
@darkled  done.

**Comment by darkled** [comment]
@tmjee I think we need to wait a bit till #1162 is merged, it has similar problem

**Comment by amihaiemil** [comment]
@asaen I am closing this; it's way too big, too many changes, and too old. I'm pretty sure it would be faster to redo the PR than fixing all the conflicts and making sure changes are up to date...

**Comment by 0crat** [comment]
The job is not in WBS, won't close the order

**Pull request closed without merging**